config/routes.rb

```diff
@@ -132,6 +132,8 @@
     end
   end
 
+  resource :changelog, only: [:show]
```

Comment:
I think this should be resources `:changelog_entries, only: [:index], path "changelog"`

Reply:
Sure. Will do.